### PR TITLE
RFC 0042 - Decommission RelEng JIRA Project in favor or Bugzilla's pr…

### PR DESCRIPTION
**Pull request by JohanLorenzo**
…oduct "Release Engineering"

[Rendered](https://github.com/JohanLorenzo/releng-rfcs/blob/bugzilla-jira/rfcs/0042-bugzilla-and-jira.md)

### CODE REVIEW

File: rfcs/0042-bugzilla-and-jira.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+For sure, we must use Bugzilla for any work related to Gecko. Leadership needs to be aware of what RelEng does through Jira.
+
+#### GitHub issues are a separate conversation
+
+GitHub issues are another bug tracking system we use. It's another split-brain situation. We agree it's conversation for later. Therefore, GitHub issues won't be part of the equation.
+
+#### Short-term planning is also a separate conversation
+
+In this RFC, we are willing to solve one problem: where should daily engineering work should be filed on? While the answer has an impact on the way RelEng tracks and plans short-term work, this is not the only parameter to take into account.
```
Comment (gabrielBusta) [inline rfcs/0042-bugzilla-and-jira.md:44]: What is short term work? How long does a project have to be to long term?

Comment (gabrielBusta) [reply rfcs/0042-bugzilla-and-jira.md:44]: Ah, I see. In the Web Platform Project Lifecycle doc it says a project is >= ~2 weeks of someone's time. So I presume short-term means < two weeks of someone's time.

File: rfcs/0042-bugzilla-and-jira.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+
+#### There won't be any synchronization between Jira and Bugzilla.
+
+[FFXP](https://mozilla-hub.atlassian.net/browse/FFXP) is for mid-to-long term planning. This Jira project doesn't have any automation in place. That shouldn't be a concern because epics on [FFXP](https://mozilla-hub.atlassian.net/browse/FFXP) don't change as often as Bugzilla bugs.
+
+
+## Open Questions
+
+* Do we want to backport the ~1000 tickets to Bugzilla?
```
Comment (gabrielBusta) [inline rfcs/0042-bugzilla-and-jira.md:94]: Leaning towards no? Sounds spam-my.

Also, the board will still be there right? Archived?

File: rfcs/0042-bugzilla-and-jira.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+* Do we want to import the currently open tickets to Bugzilla?
```
Comment (gabrielBusta) [inline rfcs/0042-bugzilla-and-jira.md:95]: Leaning yes on this one.

File: rfcs/0042-bugzilla-and-jira.md (hunk 2 above)
Comment (bhearsum) [reply rfcs/0042-bugzilla-and-jira.md:94]: I agree, let's not. Many/most of these probably already have equivalents in Bugzilla or GitHub anyways.

File: rfcs/0042-bugzilla-and-jira.md (hunk 3 above)
Comment (JohanLorenzo) [reply rfcs/0042-bugzilla-and-jira.md]: Sounds good. I added a new paragraph in the "Consequences" section

File: rfcs/0042-bugzilla-and-jira.md (hunk 2 above)
Comment (JohanLorenzo) [reply rfcs/0042-bugzilla-and-jira.md]: Fine by me. I moved this question to the "Consequences" section too.

File: rfcs/0042-bugzilla-and-jira.md (hunk 1 above)
Comment (JohanLorenzo) [reply rfcs/0042-bugzilla-and-jira.md]: You're right! I changed the paragraph to explicitly say "2 weeks or more" and "less than 2 weeks". Let me know if that looks clearer now 🙂

**Comment by JohanLorenzo** [comment]
Thanks for this first round of reviews! I clarified the wording. I'm moving this RFC to the proposal phase because reviews haven't displayed any major concerns and there are no open questions anymore.

File: rfcs/0042-bugzilla-and-jira.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+As a result, this RFC won't be on short-term planning.
+
+### Proposal
+
+#### Jira for high-level project planning
+
+Mozilla has [a process](https://docs.google.com/document/d/1BA1v2xiDF1IoqbyfY3hWlXc_MRsI02hYs2rslmkK3gM/edit#) to define which projects are more important than others. In the case of RelEng's organization, this happens within the [FFXP](https://mozilla-hub.atlassian.net/browse/FFXP) project on Jira. It’s typically the job of a manager to create Jira's epics there. The content of these epics are centered around 2 main fields: a high level description of what we plan to do and "why now?".
+
+
+#### Bugzilla for daily engineering work and implementation details
```
[the remark this answers is not in the document]
Comment (gabrielBusta) [reply rfcs/0042-bugzilla-and-jira.md]: This clears it up nicely

**Review by gabrielBusta — APPROVED** [review]

**Review by ahal — APPROVED** [review]

File: rfcs/0042-bugzilla-and-jira.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+Mozilla doesn't have the same level of quality and support when it comes to Jira.
+
+### Consequences
+
+#### [RELENG](https://mozilla-hub.atlassian.net/browse/RELENG) will be decommissioned
+
+Considering all of the above, which includes the original intent when we created [RELENG](https://mozilla-hub.atlassian.net/browse/RELENG), we shouldn't do daily engineering work on Jira anymore. Hence, [RELENG](https://mozilla-hub.atlassian.net/browse/RELENG) will be retired in favor of Bugzilla bugs under the [Release Engineering](https://bugzilla.mozilla.org/describecomponents.cgi?product=Release%20Engineering) product.
+
+
+#### Jira will still be occasionally used by engineers
```
[the remark this answers is not in the document]
Comment (JohanLorenzo) [reply rfcs/0042-bugzilla-and-jira.md:85]: Good call! As far as I know, we should interact with Jira when:
 
 * either when requesting something from another who works on Jira. 
 * or when pinged (on FFXP or another team's Jira's project).

Do you all see anything missing? If not, I'll make the next sentence more explicit.

[the remark this answers is not in the document]
Comment (bhearsum) [reply rfcs/0042-bugzilla-and-jira.md:85]: I can't think of anything. (We can always add to this later if needed.)

[the remark this answers is not in the document]
Comment (JohanLorenzo) [reply rfcs/0042-bugzilla-and-jira.md:85]: Done!

**Comment by JohanLorenzo** [comment]
Status update: the team discussed this RFC on yesterday's weekly meeting. We agreed on moving it forward and on skipping the "Final comments" phase. The main reasons are: we're all on board with the change and it only impacts Releng so we don't have to extend the RFC beyond our team.

**Pull request merged**